### PR TITLE
Added python ML packages for Cirrus GPU.

### PR DESCRIPTION
**Pull request by mbareford**

### CODE REVIEW

**Review by davidhenty — APPROVED** [review]

**Comment by aturner-epcc** [comment]
@mbareford There seem to be conflicts that need to be manually resolved before this PR can be merged. Can you resolved the conflicts and commit to the PR so I can merge

**Comment by mbareford** [comment]
@aturner-epcc It looks like I need write access to the hpc-uk build-instructions repo in order to resolve the conflict. In fact, all CSE on-duty staff would need write access to merge pull requests. Is it ok to do this?

**Pull request merged**
